Helper/Cart.php

```diff
@@ -92,7 +92,7 @@ private function add_initial_data($checkout)
     private function calculate_total()
     {
         //If discounts are allowed return cart total.
-        if ($this->config->disable_discounts !== 'yes')
+        if ($this->config->disable_discounts == 'no')
```

Comment:
No termino de comprender este fix, la comprobación vieja era mas estricta ya que solo aceptaba el flujo en caso de que estrictamente llegara el valor yes en la opcion

Reply:
Hice este cambio porque me pareció que a la hora de leer `disable_discounts' !== 'yes'` hacen un poco rebuscada la lectura, bien podria ser una comprobación de equivalencia positiva por 'no' para que a simple lectura se entienda qué sucede si no estan desactivados los descuentos.  Lo anterior se lee "si los descuentos desactivados es distinta de si", lo nuevo "si los descuentos estan desactivados"

Si es por lo estricto podriamos hacer `x === 'x'`.

De todas formas, tambien se puede volver atras tranquilamente en este cambio, no es sustancial, pero si estaria bueno que sea igual en todos lados porque eso hace que sea confuso.

Reply:
El tema es que si, por alguna razón desconocida, la opción tiene valor null o false debería también devolverse el total con `return $this->cart->get_total(null);`, por eso se estaba haciendo la comprobación contraria

Reply:
No comprendo la relación del cambio con posible fix

Reply:
vuelto atras el fix